tasks/notify.py

```diff
@@ -48,12 +50,15 @@ def check_teams(_):
 
 
 @task
-def send_message(_, notification_type="merge", print_to_stdout=False):
+def send_message(ctx, notification_type="merge", print_to_stdout=False):
```

Comment:
When we send a failure message to the author in DM, I think we should only send one message (the message that goes to the #datadog-agent-pipelines channel, which contains all failures). Here, the author will receive one message per channel where a message would be sent, which is inconvenient.

Additionally, I think we should still send the message to the #datadog-agent-pipelines channel.